apps/aecore/src/aec_block_genesis.erl

```diff
@@ -68,6 +68,8 @@ height() -> ?GENESIS_HEIGHT.
 
 miner() -> <<0:?MINER_PUB_BYTES/unit:8>>.
 
+beneficiary() -> <<0:?BENEFICIARY_PUB_BYTES/unit:8>>.
```

Comment:
Could you please update the reference to `aec_block_genesis:miner()` in [`aec_test_utils`](https://github.com/aeternity/epoch/pull/1323/files#diff-e14138de459cdd890333dfad3bd83f4cR72) even if commented out? I expect this exercise to lead to the removal of export of `aec_block_genesis:miner` and addition of export of `aec_block_genesis:beneficiary`